R/exercise.R

```diff
@@ -317,14 +419,20 @@ evaluate_exercise <- function(exercise, envir, evaluate_global_setup = FALSE) {
                   )
   )
 
+  envir_result <- duplicate_env(envir, parent = globalenv())
```

Comment:
@nischalshrestha what is your reasoning for using `globalenv()`? We think it should be:

```suggestion
  envir_result <- duplicate_env(envir)
```